.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
Please add all the files from this code linter [config](https://github.com/microverseinc/linters-config/tree/master/html-css).

Reply:
What code should I add? I copied and pasted the code from: https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml